frontend/src/global_styles/openproject/_variables.sass

```diff
@@ -4,3 +4,5 @@ $secondary-color: #bfbfbf
 
 // A selector that checks whether we are running in a test environment
 $spec-active-selector: '.env-test'
+
+$work-package--start-date-display-field-padding-left: 1.5rem
```

Comment:
Please use the same variable also for the indentation of the header element so that we don't run into problems when changing the value.